### ISSUE TITLE
Corner case in parsing major version of JDK

### ISSUE THREAD

**Issue by Eskibear** [redhat-developer/vscode-java#1331]
The [logic](https://github.com/redhat-developer/vscode-java/blob/abb5d6f1d8eba7385f979b7d6d4b1b83731711b6/src/requirements.ts#L94-L114) is to parse the string after "version", but for someone with AdoptOpenJDK 11.0.x, output of `java -version` was just as below:
```
openjdk 11.0.6 2020-01-14
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.6+10)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.6+10, mixed mode)
```
There's no "version" between "openjdk" and "11.0.6", and the major version would be recognized as 0.

Originally reported by  https://github.com/microsoft/vscode-java-pack/issues/309#issuecomment-599455779

**Comment by Eskibear** [comment]
According to https://github.com/AdoptOpenJDK/openjdk-build/issues/1610#issuecomment-601239870 , it should always have "version" there. It's not clear what was happening on the user's machine. I don't think further actions need to be taken now, let's keep an eye on it.

**Comment by fbricon** [comment]
Closed with #1635
